### ISSUE TITLE
Remove useless and empty Javadoc

### ISSUE THREAD

**Issue by jlupi** [joelittlejohn/jsonschema2pojo#659]
Hi

The generated javadoc is useless if no description is provided in the json schema.
Additionally an empty javadoc appears on properties which doesnt make sense. 

````
"via": {
   "type": "string"
},
````

````
/**
 * 
 */
private String via;

/**
 * 
 * @return
 *     The via
 */
public String getVia() {
     return via;
}

/**
 * 
 * @param via
 *     The via
 */
public void setVia(String via) {
    this.via = via;
}
````

For such cases the javadoc generation should be disabled. 

Aditionally a toggle could be added for the javadoc generation.

**Comment by joelittlejohn** [comment]
Yes agreed. I feel like this is a problem that hasn't always existed.

Let's definitely sort this out for the next version. I'm not interested in adding a config property for this, the javadoc just shouldn't be added if there is no useful information.

**Comment by joelittlejohn** [comment]
It looks like the empty javadoc has been introduced for all fields by #645.

**Comment by joelittlejohn** [comment]
The `@param` and `@return` were added to avoid Javadoc warnings in Java 8 by 6e5ace79ce68459fa3a598b6445374324e78e8b9. I would prefer to remove these as this is useless noise in the source code.

**Comment by jlupi** [comment]
@joelittlejohn I fully agree. Thanks.

**Comment by annafw** [comment]
Without the javadoc being generated, there are warnings in Java 8 for missing @return and @param docs. What's the workaround for that?

**Comment by joelittlejohn** [comment]
@annawinkler Hopefully this link will help: http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html

**Comment by annawinkler** [comment]
That helped - thanks!

Anna
